frank/geometry.py

```diff
@@ -313,6 +328,15 @@ def fit(self, u, v, V, weights):
         self._dRA = dRA
         self._dDec = dDec
 
+        logging.info('    Time taken for geometry %.1f sec' %
+                     (time.time() - t1))
+
+        logging.info('    Using: inc  = {:.2f} deg,\n           PA   = {:.2f} deg,\n'
+                     '           dRA  = {:.2e} mas,\n'
+                     '           dDec = {:.2e} mas'.format(self._inc, self._PA,
+                                                           self._dRA*1e3,
+                                                           self._dDec*1e3))
+
```

Comment:
I think the timing and printing of the geometry belongs in fit.py There is no need to print the results by default since they are stored and easily accessible in future code. 

If you *really* want to make it easy for a user to see the geometry values, add a __str__ method that returns this as a string. Then if you call print(geometry) you'll get a print out of the geometry.

Reply:
I moved these to fit.py and am leaving printing of the used values as default there because if running from the terminal, the point is to give the user a quick sense of whether the geometry values are reasonable, not to ask them to wait for the fit to end and then open a python terminal to load in and print the geometry values.

Reply:
I'm ok with them being default if they are in fit.py, but not here.

Reply:
Moved to fit.py